mantidimaging/gui/widgets/mi_mini_image_view/view.py

```diff
@@ -36,12 +36,22 @@ class MIMiniImageView(GraphicsLayout, BadDataOverlay, AutoColorMenu):
     bright_levels: None | list[int] = None
     levels: list[float]
 
-    def __init__(self, name: str = "MIMiniImageView", parent: QWidget | None = None, recon_mode: bool = False):
+    def __init__(self,
+                 name: str = "MIMiniImageView",
+                 parent: QWidget | None = None,
+                 recon_mode: bool = False,
+                 view_box: ViewBox | None = None):
```

Comment:
Could we make the caller pass in the type rather than an instance. e.g 
`view_box_type: type[ViewBox] = ViewBox`

When the `MIMiniImageView` is created, we pass in the type like
`self.image = MIMiniImageView(name="Projection", view_box_type=CustomViewBox)`